modules/gatk4/variantrecalibrator/meta.yml

```diff
@@ -0,0 +1,92 @@
+name: gatk4_variantrecalibrator
+description: |
+  Build a recalibration model to score variant quality for filtering purposes.
```

Comment:
we might want to add here in description that this module should be used according to best practices, i.e. when sufficient data are present for the gaussian mixture model to be applied (recommended for exome: 30 samples).